cmd/sidecar_mounter/gcsfuse_binary

```diff
@@ -1 +1 @@
-gs://gke-release-staging/gcsfuse/v2.3.1-gke.0/gcsfuse_bin
+gs://gke-gcsfuse-test-binary
```

Comment:
This will be removed when the new gcsfuse release includes the prometheus changes.